staging/staging.mk

```diff
@@ -0,0 +1,51 @@
+# staging.mk
+# this makefile is being included by staging/Makefile.include and staging/Makefile.dep.
+
+#
+# list of staging modules
+#
+# In order to mark a module as broken, add it to the "BROKEN" variable.
+# That way, it's dependees will be marked broken implicitly.
+#
+#STAGING_MODULES += test
+#BROKEN += test    # (broken due to change in X, see #12345)
+
+#
+# list of stating applications
+#
+# In order to mark a staging application as "broken", turn it's
+# line into a comment (and add a note).
+#
+#STAGING_APPS += broken/application/example # (intentionally broken by design)
```

Comment:
This currently does not build *hello-staging*.

Reply:
fixed